php_phongo.c

```diff
@@ -2907,8 +2917,8 @@ static mongoc_client_encryption_opts_t* phongo_clientencryption_opts_from_zval(m
 		zval*  kms_providers  = php_array_fetchc(options, "kmsProviders");
 		bson_t bson_providers = BSON_INITIALIZER;
 
-		if (Z_TYPE_P(kms_providers) != IS_ARRAY) {
-			phongo_throw_exception(PHONGO_ERROR_INVALID_ARGUMENT TSRMLS_CC, "Expected \"kmsProviders\" encryption option to be an array");
+		if (Z_TYPE_P(kms_providers) != IS_ARRAY && Z_TYPE_P(kms_providers) != IS_OBJECT) {
```

Comment:
Ah, I forgot we were already checking for an array type here. Good to know both instances of "kmsProviders" handling are consistent now!